### DISCUSSION TITLE
Explorer option missed in which key?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1848] [Q&A] [answered]
You can see in the image that I have no explorer relate options.  I havent change which key option in my config. I remove all keymap with leader key. But not work how to solve this question?

> press leader
![image](https://github.com/LazyVim/LazyVim/assets/11295355/2f2182df-0bd3-4095-932b-ba06a08e72cc)
> press leader + f
![image](https://github.com/LazyVim/LazyVim/assets/11295355/f7ddd27b-cb44-42b1-afa7-80cc5b5c344f)

---
related code I found in local/share/nvim. But I never change the code here.

```
> ag Explorer
lazy/LazyVim/lua/lazyvim/plugins/editor.lua
16:        desc = "Explorer NeoTree (root dir)",
23:        desc = "Explorer NeoTree (cwd)",
25:      { "<leader>e", "<leader>fe", desc = "Explorer NeoTree (root dir)", remap = true },
26:      { "<leader>E", "<leader>fE", desc = "Explorer NeoTree (cwd)", remap = true },
```

**Comment by dpetka2001** [comment, marked as answer]
Did you remove `neo-tree` plugin?

**Reply by OriLiMu** [reply]
After I reboot the pc, it come back. So weird. Thank you
